tests/test-travis/test-revcheck.R

```diff
@@ -0,0 +1,8 @@
+library(testit)
+
+assert("check_deps() works", {
+  x <- check_deps("xfun")
+  (is.character(x$check))
+  (is.character(x$install))
```

Comment:
I feel that testing if the result is character is not very meaningful (it's fine if the result is NULL, which is not character). I'd rather find other functions to test, e.g., `clean_Rcheck()`.